test/tests/functional/pbs_hooksmoketest.py

```diff
@@ -19,25 +19,21 @@
 from tests.functional import *
 
 
-@tags('hooks')
+@tags('hooks', 'smoke')
 class TestHookSmokeTest(TestFunctional):
-
     """
     Hooks Smoke Test
     """
-
     def setUp(self):
         TestFunctional.setUp(self)
-        self.server.manager(MGR_CMD_SET, SERVER, {'log_events': 2047})
-        self.server.manager(MGR_CMD_SET, SERVER,
-                            {'scheduling': 'False'})
+        a = {'log_events': 2047, 'scheduling': 'False'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
         self.clean_hooks()
 
-    def get_file_path(self, hook_name):
+    def get_file_paths(self, hook_name, existence=False):
```

Comment:
now, the function does the job of checking the .hk file, so we should name it accordingly.